usecase/output/user.go

```diff
@@ -8,10 +8,13 @@ type ListUsers struct {
 
 type UserDTO struct {
 	ID            string     `json:"id"`
-	LoginID       string     `json:"loginID"`
+	ShopID        string     `json:"shop_id"`
```

Comment:
[must]
キャメルケースでお願いします！
```suggestion
	ShopID        string     `json:"shopID"`
```